Sources/LiveViewNative/Views/Controls and Indicators/Pickers/Picker.swift

```diff
@@ -27,6 +27,9 @@ import SwiftUI
 /// ## Attributes
 /// - ``selection``
 ///
+/// ## Attributes
+/// - ``selection``
+///
```

Comment:
This may have been merged wrong, since the attributes section is duplicated here. The commit message of [8ed9dd3](https://github.com/liveview-native/liveview-client-swiftui/pull/1115/commits/8ed9dd395c9948f2172438afaab8a3220d72eff0) matches [bc73079](https://github.com/liveview-native/liveview-client-swiftui/commit/bc73079abb730eed5562a13ddabad4a13b63061a) which is on `main`.

Reply:
Should be able to do an interactive rebase and drop [8ed9dd3](https://github.com/liveview-native/liveview-client-swiftui/pull/1115/commits/8ed9dd395c9948f2172438afaab8a3220d72eff0). All it added was this doc comment.

Reply:
@carson-katri I was running into an issue with interactive rebase, so I just removed it in a subsequent commit.